react/react2/week1/my-app/src/GithubUsers.js

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from "react";
+import "./Githubsearch.css";
+import { SearchInput } from "./SearchInput";
+
+export function GithubUsers() {
+  const [apiData, setApiData] = useState("");
```

Comment:
```suggestion
  const [users, setUsers] = useState([]);
```

Similarly here, this component should not care where the data comes from. That is an implementation detail, try to name your variables in the solution domain, not the code domain. This is the users you need in order to display a list of users.

Further, try to give an appropriate starting value to `useState`, if you know it will be a list, then make the initial value an empty list.